content/nextcloud.md

```diff
@@ -232,22 +257,11 @@ chown -R www-data:www-data /var/www/nextcloud
 chmod -R 755 /var/www/nextcloud
 ```
 
-
-Start and enable the php-fpm and the mariadb services (the name of the
-php-fpm service may have a version number ahead of it, use bash\'s tab
-autocomplete to help you out with that):
-
-```sh
-systemctl enable php7.4-fpm
-systemctl start php7.4-fpm
-systemctl enable mariadb
-systemctl start mariadb
-```
-
-Reload the nginx service:
+Start and enable the php-fpm and the nginx service:
 
 ```sh
-systemctl reload nginx
+systemctl enable php7.4-fpm --now
+systemctl enable nginx --now
```

Comment:
This guide assumes Nginx is already installed.  `enable --now` won't reload nginx if it's already started, so a `reload` is more appropriate here.  

Additionally, mentioning the nginx guide from the top of this one might be a good idea.  Something to the effect of "you need a working [nginx server] before following this guide"

Reply:
Fixed, thanks!